syft/dependency_check.py

```diff
@@ -1,10 +1,33 @@
+from __future__ import absolute_import
```

Comment:
the same here if we do not care about python2

Reply:
true, fixed, thanks!